### PR TITLE
Flasher.js Getting Started

### PR DESCRIPTION
**Pull request by samjulien**
First crack at #1 to get folks up and running on Flasher.js. Fixed some typos and errors I found so re-submitting.

### CODE REVIEW

File: flasher.js/getting_started.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Let's go!
...
+First things first: head over to the [Flasher.js releases page](https://github.com/thingsSDK/flasher.js/releases) and download the latest version. We have lots of options for you - you can run Flasher.js on Linux, Mac, and Windows (both 32bit and 64bit). You can even compile the source yourself if you're feeling brave. 
 
-On Windows, you'll be downloading the binary directly, so you can just move it where you'd like it. On a Mac, you'll be downloading a disk image (.dmg) file. Double-click it and drag it over to your Applications folder. If you're on Linux you'll be downloading a Debian package (.deb). Double-click it to install.
+On Windows, you'll be downloading the binary directly, so you can just move it where you'd like it. On a Mac, you'll be downloading a disk image (.dmg) file. Double-click it and drag the Flasher.js icon over to your Applications folder. If you're on Linux, you'll be downloading a Debian package (.deb). Double-click it to install.
```
Comment (chalkers) [inline flasher.js/getting_started.md]: The binary, for Windows, is an installer that will create a shortcut in your start menu.

**Review by chalkers — CHANGES_REQUESTED** [review]
The line about "binary directly" could be misconstrued to mean that it's just the plain-old-binary not an installer.

**Comment by chalkers** [comment]
Thanks @samjulien - I tweaked the line :)

**Comment by samjulien** [comment]
Ah thanks, I had a busy afternoon and evening.

**Pull request merged**